### ISSUE TITLE
Cannot use `Balance` and `Swap` component on localhost.

### ISSUE THREAD

**Issue by lukema95** [zeta-chain/universalkit#32]
I use Universalkit's Swap and Balance components in template:
```ts
const Page = () => {
  return (
    <div className="m-4">
      <div className="flex justify-end gap-2 mb-10">
        <ConnectBitcoin />
        <ConnectButton label="Connect EVM" showBalance={false} />
      </div>
      <div className="flex justify-center">
        <div className="w-[400px]">
          <Swap contract={contract} />
          <Balances />
        </div>
      </div>
    </div>
  );
};
```
Fontend:
<img width="494" alt="image" src="https://github.com/user-attachments/assets/d84762e5-56a8-4474-827d-42a310bc8029" />

Got error:
<img width="326" alt="image" src="https://github.com/user-attachments/assets/51150429-2260-4626-94c9-a06609a7de4a" />
<img width="318" alt="image" src="https://github.com/user-attachments/assets/10b4201c-8393-47b1-b579-df0645cee11b" />

I used "@zetachain/universalkit": "^2.0.0.

**Comment by fadeev** [comment]
Yes, these components weren't built with localnet in mind, because they make queries to other chains.

**Comment by lukema95** [comment]
> Yes, these components weren't built with localnet in mind, because they make queries to other chains.

I'm connected to a test network. I just started the service locally. This appears to be a CORS issue causing the API access request to be denied

**Comment by lukema95** [comment]
> > Yes, these components weren't built with localnet in mind, because they make queries to other chains.
> 
> I'm connected to a test network. I just started the service locally. This appears to be a CORS issue causing the API access request to be denied

@fadeev After I replaced the rpc of blockpi, it can work normally. I guess blockpi has some limitations at present, so we can consider not using it as the default rpc

**Comment by fadeev** [comment]
@lukema95 yes, I've been having issues with BlockPi lately, too.
